index.html

```diff
@@ -198,6 +198,53 @@ <h3>Skills</h3>
           </ul>
         </div>
       </section>
+
+      <form action="https://formspree.io/f/xoqbveqq" method="post">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h2>
+        <ul class="form-ul">
+          <li class="form-li">
+            <input autocomplete="off" maxlength="30" type="text" name="name" id="name" placeholder="Full Name" required>
+          </li>
+          <li class="form-li">
+            <input autocomplete="off" type="email" name="email-address" id="email-address" placeholder="Email address" required>
+          </li>
+          <li class="form-li">
+            <textarea autocomplete="off" name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] Please add an attribute `required` to your message input to ensure your form is not submitted with an empty message input field